### PR TITLE
Lesson14 arrays&loops

### PR DESCRIPTION
**Pull request by Laraib1121**

### CODE REVIEW

File: 14 Arrays and Loops/ArraysAndLoops.playground/Pages/Adding Items.xcplaygroundpage/Contents.swift [hunk 1]
```diff
@@ -5,20 +5,20 @@
  
  Remember from the Instances playground that a type followed by parentheses is how you create an instance of that type. To create a mutable empty array that will hold strings, do this:
 */
-var list = [String]()
+var emptyList = [String]()
```
Comment (RndmTsk) [inline 14 Arrays and Loops/ArraysAndLoops.playground/Pages/Adding Items.xcplaygroundpage/Contents.swift:8]: You can also do this:
```swift
var emptyList: [String] = []
```

File: 14 Arrays and Loops/ArraysAndLoops.playground/Pages/Exercise-Screening Messages.xcplaygroundpage/Contents.swift [hunk 2]
```diff
@@ -22,7 +22,12 @@ If the message refers to the Caterpillar, print it to the console.
  The `contains` method is part of the `Foundation` framework that you read about in the “Types” playground. If you try using it and get an error saying “Value of type 'String' has no member 'contains',” follow the instructions from that playground to import the framework into your project.
  */
 // Write the `for…in` loop here:
-
+for message in aliceMessages {
+    if message.contains("Caterpillar"){
```
Comment (RndmTsk) [inline 14 Arrays and Loops/ArraysAndLoops.playground/Pages/Exercise-Screening Messages.xcplaygroundpage/Contents.swift:26]: Another option:
```swift
for message in aliceMessage {
    guard message.contains("Caterpillar") else { continue }
    print(message)
}
```

**Review by RndmTsk — APPROVED** [review]
Looks good, hopefully you're becoming a bit more comfortable with arrays and loops now!

File: 14 Arrays and Loops/ArraysAndLoops.playground/Pages/Adding Items.xcplaygroundpage/Contents.swift (hunk 1 above)
Comment (Laraib1121) [reply 14 Arrays and Loops/ArraysAndLoops.playground/Pages/Adding Items.xcplaygroundpage/Contents.swift:8]: alright ill make note of it

File: 14 Arrays and Loops/ArraysAndLoops.playground/Pages/Exercise-Screening Messages.xcplaygroundpage/Contents.swift (hunk 2 above)
Comment (Laraib1121) [reply 14 Arrays and Loops/ArraysAndLoops.playground/Pages/Exercise-Screening Messages.xcplaygroundpage/Contents.swift:26]: what does guard mean?

Comment (RndmTsk) [reply 14 Arrays and Loops/ArraysAndLoops.playground/Pages/Exercise-Screening Messages.xcplaygroundpage/Contents.swift:26]: I wasn't sure if you'd gotten to it yet, but it's like an `if` that forces you to early return. If it doesn't make sense to you, just take a note of it and we can cover it at our next meeting.

**Comment by RndmTsk** [comment]
Looks good, feel free to merge.

**Pull request merged**